extensions/amp-experiment/0.1/test/test-amp-experiment.js

```diff
@@ -25,12 +25,14 @@ describe('amp-experiment', () => {
 
   const config = {
     'experiment-1': {
+      grouping: 'xyz',
```

Comment:
They're actually not needed in this test anymore, since the logic has been moved to variant.js.

Removed.